packages/xo-web/src/xo-app/new/sr/index.js

```diff
@@ -356,6 +356,18 @@ export default class New extends Component {
     }
 
     try {
+      // local SR
+      if (type === 'ext' || type === 'lvm' || type === 'zfs' || type === 'local') {
```

Comment:
Only `ext` and `lvm` are concerned at the moment. And `xfs` but it's not supported in the UI yet (maybe in the future). The `zfs` driver does not format any device, nor does the local ISO driver IIRC. @Wescoeur should be able to confirm.

General comment: wouldn't it be more future proof to create classes or objects that describe the SR type, where you define an attribute that means that sr-create is destructive, so that each time you add a new SR you have no risk of forgetting to update this `if` that you will probably have forgotten about anyway?

Reply:
Yes that would be best, let's do it here if trivial, otherwise, it should be part of another PR.